changelog: add entry for team-level and platform-level AI feature opt-out (#5056)
FlowFuse/website: src/changelog/2026/05/ai-opt-out.md

---
title: AI Feature Opt-Out for Teams                                                                                                                                                                                                                                                                                         

> description: Team owners and platform admins can now disable all AI features at the team level, giving full control over AI functionality across the platform.                                                                                                                                                              

Reviewer: ```suggestion
description: Team owners and platform admins can now disable all AI features at the team level, so they can turn AI off when policy or compliance requires it.                                                                                                                                                              
```

Reviewer: concrete reason why is better than an abstract

authors: ['serban-costin']                                                                                                                                                                                                                                                                                      
tags:                                                                                                                                                                                                                                                                                                                       
- changelog                                                                                                                                                                                                                                                                                                             
issues:
- https://github.com/FlowFuse/flowfuse/issues/7316
---

> You can now turn off AI features for your team from the team settings page. When disabled, the Expert Assistant, inline code completions, snapshot description generation, and the nr-assistant plugin are all removed for your team. Running instances will need a restart for the change to fully take effect.            

Reviewer: ```suggestion
You can now turn off AI features for your team from the team settings page. When disabled, FlowFuse removes the Expert Assistant, inline code completions, and snapshot description generation for your team. Running instances will need a restart for the change to fully take effect.            
```

Reviewer: passive to active voice

![AI Features toggle in team settings](./images/ai-opt-out-toggle.png){data-zoomable}                                                                                                                                                                                                                                       
*Opt out of AI features from the Danger section in team settings.*

The toggle is on by default for all teams. Flipping it off shows a confirmation dialog so you don't accidentally disable it. If you change your mind later, just flip it back on.

> Self Hosted Enterprise customers also get a platform-level and team-type-level AI feature flag, so admins can control AI availability across the entire installation or per plan before it ever reaches team owners.

Reviewer: ```suggestion
Self Hosted Enterprise customers also get platform-level and team-type-level AI controls, so admins can control AI availability across the entire installation or per plan before it ever reaches team owners.
```

Reviewer: "feature flag" counts as tech jargon and we try to avoid that
